src/util/qrdisplay.js

```diff
@@ -6,12 +6,14 @@ const getQRDataURI = (data) => {
   return 'data:image/png;charset=utf-8;base64, ' + pngBuffer.toString('base64')
 }
 
-const openQr = (data) => {
+const openQr = (data, stop) => {
```

Comment:
@zachferland I can see how in our current implementation of the UI the stop here is required, but I don't like the complexity, as it's not needed for other a typical redux application for example. Leave it in for now, but put a comment in the code that it may be removed in the future.

Leave it in for now, but make it